java/src/main/java/ai/rapids/cudf/CuFileWriter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.rapids.cudf;
+
+/**
+ * Represents a cuFile file handle for reading.
+ */
+public final class CuFileWriter extends CuFileHandle {
```

Comment:
Similar naming thing here
```suggestion
public final class CuFileWriteHandle extends CuFileHandle {
```

Reply:
Done.